movies-search/src/Search/Search.jsx

```diff
@@ -20,10 +21,16 @@ class Search extends React.Component {
     });
   }
 
+  handleSearch = (e) => {
+    if (e.keyCode == 13) {
+      this.props.fetchData(`/search/search=${this.state.input}&searchBy=${this.state.searchField.toLowerCase()}`)
```

Comment:
in real apps you won't be mixing component logic with routes. So here, if you have time, extract the http logic into a separate service